src/ex1.py

```diff
@@ -0,0 +1,20 @@
+def get_week_day (week: int) -> int:
```

Comment:
attenzione, hai indicato che questa funzione ritorna un intero ma in realtà ritorna una stringa :)